Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bevy_tile_atlas"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Normally I prefer updating the version myself in a separate commit for clarity and in case I need to tack anything else on.

If you do this, could you also update the two usages in the `README.md` file?

Reply:
I reverted the version number change.